src/Symfony/Component/Notifier/Tests/Exception/UnsupportedSchemeExceptionTest.php

```diff
@@ -89,6 +90,7 @@ public static function setUpBeforeClass(): void
             SmsBiurasTransportFactory::class => false,
             SpotHitTransportFactory::class => false,
             TelegramTransportFactory::class => false,
+            TelnyxTransportFactory::class => false,
             TwilioTransportFactory::class => false,
             ZulipTransportFactory::class => false,
         ]);
```

Comment:
You need to adjust the provider as well please

Reply:
Updated